pyserini/util.py

```diff
@@ -71,6 +97,10 @@ def download_url(url, save_dir, md5=None, force=False, verbose=True):
     if md5:
         assert compute_md5(destination_path) == md5, f'{destination_path} does not match checksum!'
 
+def get_cache_home():
+    return os.path.expanduser(
+        os.getenv("PYS_HOME", os.path.join(os.getenv("XDG_CACHE_HOME", f'~{os.path.sep}.cache'), "pyserini"))
```

Comment:
Are `PYS_HOME` and `XDG_CACHE_HOME` commonly used?

What about something simpler like https://stackoverflow.com/questions/4028904/how-to-get-the-home-directory-in-python